app/src/main/java/com/example/ubclaunchpad/launchnote/toolbar/PhotoNavigatonToolbarFragment.kt

```diff
@@ -0,0 +1,130 @@
+package com.example.ubclaunchpad.launchnote.toolbar
+
+import android.content.Context
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+
+import com.example.ubclaunchpad.launchnote.R
+import java.io.Serializable
+import kotlin.collections.ArrayList
+
+/**
+ * A simple [Fragment] subclass.
+ * Activities that contain this fragment must implement the
+ * [PhotoNavigatonToolbarFragment.OnButtonPressListener] interface
+ * to handle interaction events.
+ * Use the [PhotoNavigatonToolbarFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class PhotoNavigatonToolbarFragment : Fragment() {
+
+    // TODO: Rename and change types of parameters
+    private var mMode: ToolbarMode = ToolbarMode.NormalMode
+    private var mListener: OnButtonPressListener? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        if (arguments != null) {
```

Comment:
you can do `arguments?.let { mMode = arguments.getSerializable(MODE) as ToolbarMode }`
`?.let` is basically a more concise null check.

Reply:
sure